packages/desktop/Cargo.toml

```diff
@@ -36,6 +36,8 @@ tracing = "0.1"
 tracing-subscriber = { version = "0.3", features = ["env-filter"] }
 netdev = "0.24"
 regex = "1"
+window-vibrancy = "0.5.2"
+csscolorparser = "0.7.0"
```

Comment:
think it'd be best if we got rid of `csscolorparser` and stored the color in whatever format is easiest to parse. especially since these'll get added to the settings UI, it doesn't matter as much to make it user-friendly to edit the color string